### PR TITLE
Bash and zsh documentation

### PR DESCRIPTION
**Pull request by JosueOb**
# IOET LEARNING PATH

## Authors

- @dsgarcia8 
- @PieritoAlva95 
- @JosueOb 
- @Jobzi 
- @JuanJosephG 

## Description

This PR related to bash and zsh documentation.

## Checklist

- [x] This PR is ready for review.
- [x] This PR has been self-reviewed (for code styling, misspellings, etc.) before assigning reviewers.
- [x] This PR fulfills all the acceptance criteria.
- [x] Documentation has been created/updated, as needed.
- [x] Authors have been added correctly.

### CODE REVIEW

File: 1. Developer basics/5_shell_scripting/5-1_bash_shell_zsh.md [hunk 1]
```diff
@@ -45,9 +66,231 @@ For the most part bash and zsh are almost identical which is a relief. Navigatio
...
 | Smaller community                                                                             | Wider community                                                               |
 
-<br />
+## How to install and set up Zsh as default
+
+Installation on many platforms and setting up Zsh as default you can
+find [here](https://github.com/ohmyzsh/ohmyzsh/wiki/Installing-ZSH#install-and-set-up-zsh-as-default)
+.
+
+## Contents
```
Comment (Wason1797) [inline 1. Developer basics/5_shell_scripting/5-1_bash_shell_zsh.md]: Maybe we can have the contents before the intro and how to install?

**Review by Wason1797 — CHANGES_REQUESTED** [review]
Looks great! just one minor comment to address.

File: 1. Developer basics/5_shell_scripting/5-1_bash_shell_zsh.md (hunk 1 above)
Comment (JosueOb) [reply 1. Developer basics/5_shell_scripting/5-1_bash_shell_zsh.md]: Change done! @Wason1797

**Review by Wason1797 — APPROVED** [review]
Looks great!

**Pull request merged**